core-lightning/umbrel-app.yml

```diff
@@ -32,33 +32,27 @@ defaultPassword: ""
 submitter: Blockstream
 submission: https://github.com/getumbrel/umbrel-apps/pull/475
 releaseNotes: >-
-  This update adds support for Backups in umbrelOS 1.5.
+  This updates cln-application to v25.07.3, lightningd to v25.09.2, and replaces the legacy standalone c-lightning-rest service with the built-in REST interface of lightningd.
```

Comment:
@ShahanaFarooqui can you please doublecheck that you are okay with this wording change I've made for c-lightning-rest?

I changed

> ...and removes c-lightning-rest

to 

> ...and replaces the legacy c-lightning-rest service with the built-in REST interface of lightningd.

Just to avoid anyone reaching out thinking theres no rest capabilities at all anymore

Reply:
thanks